Gemfile

```diff
@@ -103,7 +103,7 @@ gem 'swagger-blocks'
 gem 'typhoeus'
 gem 'upsert'
 gem 'utf8-cleaner'
-gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', branch: 'master'
+gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', tag: '3.140.0'
```

Comment:
Can we reverse this?  Typically users don't create tags/releases.
Typically the way to update is the version is to just run `bundle update vets_json_schema`.
That'll get you the same result here (i.e. version 3.140.0 will be specified in Gemfile.lock)